### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by jerrahall** [Chia-Network/chia-blockchain#3175] [General]
Oi tenho um mini pc com essas especificações 
4 gb de ram. ( expancivel ate 8gb )
Core 2duo e 7500 ( pode troca )
Gt 9500 ( pode troca também e originalmente veio com uma gt 1030 2gb mais deu poblema ) 
Hd 500 gb ssd 120 gb 
Windows 7 original 
E pelo que vi não roda o programa 
Na hora de iniciar a "mineração" aparece o primeiro erro ( Caught plotting error: Bitfield plotting not supported by CPU : -(32-INFO [6884] Failed to execute script chia ) e pelo que entendi meu processador não suporta o programa ou a "mineração" e logo em seguida aparece
 o erro unknown error 

Alguém sabe um processador que não dê esse erro tenho outro processador um core2quad q9400 será que vai dá esse erro também por causa disso não comecei ainda

**Comment by rfmendes** [comment]
Eu também estou com o mesmo problema! No chia.net nos requisitos de processador pede:

Quad core 1.5Ghz CPU
2 GB Ram
Python 3.7 and above

Estou com um processador "Core 2 Quad de 2.4Ghz" e também ocorre este erro e fica parado no 1% 

:(

**Comment by dorkmo** [comment]
processors made before 2010 dont support bitfield plotting. you can disable bitfield plotting and it should still work. but i dont know if you have enough ram to plot? you might install ubuntu server on a usb stick and mine that way

**Comment by rfmendes** [comment]
How to disable this Bitfield function?

How much will I need RAM?

**Comment by dorkmo** [comment]
https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference
-e [bitfield plotting]: Using the -e flag will disable the bitfield plotting algorithm

https://www.chia.net/2021/02/22/plotting-basics.html
1400

keep in mind you also need enough ram to run your operating system. so it might be best to run ubuntu server edition

**Comment by rfmendes** [comment]
OK! TKS 
:)
